engineapi/types.go

```diff
@@ -13,7 +13,7 @@ import (
 const (
 	// CurrentCLIVersion indicates the default API version manager used to talk with the
 	// engine, including `longhorn-engine` and `longhorn-instance-manager`
-	CurrentCLIVersion = 6
+	CurrentCLIVersion = 7
```

Comment:
We missed to bump this in 1.4.0?.. It seems fine, because the minVersion of engine did not change. https://github.com/longhorn/longhorn-engine/blob/67b890eb3a1ce253d109cd5546d9ce17fe27cc3e/pkg/meta/version.go#L6-L6

Reply:
Yes... Since and min version does not change, everything is still compatible.